### ISSUE TITLE
Support for Laravel 5.7

### ISSUE THREAD

**Issue by misog** [LaravelDaily/quickadmin#140]
Hello will there be support for Laravel 5.7?

**Comment by PovilasKorop** [comment]
@misog yes scheduled next week.

**Comment by sungil** [comment]
Why don't you upgrade to 5.8?

**Comment by PovilasKorop** [comment]
@sungil good point, totally forgot, we don't really support this package actively anymore. But ok adding to the to-do list to upgrade to 5.8 next week.
